internal/pkg/otel/testdata/all-components.windows.yml

```diff
@@ -0,0 +1,3 @@
+exporters:
+  file:
+    path: ${env:TMP}\file-exporter-output.json
```

Comment:
seems that windows supports a lot less? is this from upstream or just how we have enabled it?